cpp/proto/serializer.h

```diff
@@ -368,28 +400,6 @@ class Deserializer {
     return DeserializeResult::OK;
   }
 
- private:
```

Comment:
I think I dinged this `private:` by mistake?!? Oh, yeah, I meant for this to be in this commit, put it in the wrong one:

https://github.com/pphaneuf/certificate-transparency/commit/136e12edebd44c06d3c25dac046210e114faf543#diff-4ae97ce672fdbd80f64b37442df606d4R345

There's a similar thing for `Serializer` above, too:

https://github.com/pphaneuf/certificate-transparency/commit/136e12edebd44c06d3c25dac046210e114faf543#diff-4ae97ce672fdbd80f64b37442df606d4R258